doc/tutorial.txt

```diff
@@ -258,6 +258,15 @@ inherit the affinity from the task level):
 	}
 }
 
+*** taskgroups
+
+* taskgroup: String. Can be specified at task level or phase level. Currently
+only SCHED_OTHER tasks are supported to contain taskgroup data. Tasks with a
+policy other than SCHED_OTHER and without taskgroup data can coexist in the
+setup. An empty taskgroup ("") is allowed and is interpreted as if there is
+no taskgroup data given. Pre-existing parts of a taskgroup are preserved.
+They are not deleted during the clean-up phase.
+
```

Comment:
This is documented as part of the policy section, distinct from the event section so it's consistent. What do you think about having this be an event 'change taskgroup' instead?

Reply:
Like I said, I would keep taskgroup ("taskgroup") like cpuset ("cpus") or sched_data ("policy"/"priority").

Reply:
I also feel that we should treat cgroup changes like policy/affinity changes.

Reply:
OK, that's good for me - I think we made a mistake originally in having a difference between policy and events, but it is done now and nobody would really change that. In that context, I agree that this is much more similar to a sched policy/cpuset change.

Reply:
@credp I agree that we might have done a mistake when we decided to not treat cpuset and sched policy like normal event and this would have enable more flexibility in the scenario. But it's worth keeping task group like cpuset/policy and we can change the behavior of theses aspart of another dev